var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -138,6 +138,18 @@ class Gaudi(CMakePackage):
     # The Intel VTune dependency is taken aside because it requires a license
     depends_on("intel-parallel-studio -mpi +vtune", when="+vtune")
 
+    # /usr/bin/env triggers SIP, but we still need the LD_LIBRARY_PATH for gaudirun.py
+    # hardcode the absolute path instead
+    @when(sys.platform == "darwin")
```

Comment:
```suggestion
    @when("platform=darwin")
```
See e.g. emacs, hpcviewer.